stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerInitializer.kt

```diff
@@ -129,6 +130,28 @@ internal class DefaultFlowControllerInitializer(
         )
     }
 
+    private suspend fun setLastSavedPaymentMethod(
+        prefsRepository: PrefsRepository,
+        isGooglePayReady: Boolean,
+        paymentMethods: List<PaymentMethod>
```

Comment:
The way you have it is totally fine, but you could alternatively change the method signature to be as shown below and pass in `paymentMethods.firstOrNull()` as the 3rd argument. This might help explain what this payment method represents.

```
 private suspend fun setLastSavedPaymentMethod(
        prefsRepository: PrefsRepository,
        isGooglePayReady: Boolean,
        mostRecentPaymentMethod: PaymentMethod?
)
```